style.css

```diff
@@ -314,9 +293,6 @@
   .list-work {
     width: 80%;
     border-right-width: 20%;
-  }
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But the dot divider is not present in all your project card so please make sure that you fix it :pray: 
  
  | Your Design |Figma Design |
  | :---:   | :---: | 
  | ![image](https://user-images.githubusercontent.com/92303891/235319041-335ab0a2-a123-4dfc-a3b4-e870ed2a9008.png) | ![image](https://user-images.githubusercontent.com/92303891/235318851-768b5905-0cf8-4164-ba75-c6e680f19dde.png) |
  

- [ ] Also as you can see, all your project title's colors and and the company (for instance the "Canopy") color does not match the Figma design so make sure that you fix them